migrations/migrations/20181025175848-albums.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('Purchases', {
+      id: {
+        type: Sequelize.INTEGER,
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        unique: true
+      },
+      userId: {
+        type: Sequelize.INTEGER,
+        foreignKey: true,
+        allowNull: false
+      },
+      albumId: {
```

Comment:
usually the convention is that the db fields go in `snake_case` but js like stuff in `camelCase`, so you can leave here `album_id` and in the model leave the attribute `albumId` but with the configuration `field: 'album_id'` to bind both field and attribute

Reply:
Fixed, but couldn't link userId to the user_id column in the foreign key column